components/01-globals/typography/Heading.vue

```diff
@@ -1,3 +1,12 @@
+<template>
+  <component
+    :is="tag ? tag : 'h' + level"
+    :class="[tag && `heading heading--${levelClassName}-level`,
```

Comment:
Why do you check `tag` if you need `levelClassName`?

Reply:
B/c when we have a tag, e.g. `h6,` in styles in tag we extend `heading` and `heading--${levelClassName}-level`. It does not make sense to add classes, when we extend this styles inside the tag. If we want to add this classes, I think, we should remove a part of styles. :)